api/external/cfgmgmt/cfgmgmt.proto

```diff
@@ -120,6 +120,24 @@ service ConfigMgmt {
     option (chef.automate.api.iam.policy).action = "infra:nodes:list";
   };
 
+  /*
+  GetCheckinCountsTimeSeries
+
+  Returns a daily time series of unique node check-ins for the number of day requested
```

Comment:
day -> days

maybe add a comment about "if days ago value is empty, api will return the default 1 day ago results" ?